src/main/java/net/sf/jabref/sql/DatabaseType.java

```diff
@@ -0,0 +1,40 @@
+package net.sf.jabref.sql;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * All DBTypes must appear here. The enum items must be the
+ * names that appear in the combobox used to select the DB,
+ * because this text is used to choose which DatabaseImporter/Exporter
+ * will be sent back to the requester
+ */
+public enum DatabaseType {
+    MYSQL("MySQL"), POSTGRESQL("PostgreSQL");
+
+    private final String serverType;
+
+    DatabaseType(String dbType) {
+        this.serverType = dbType;
+    }
+
+    public String getServerType() {
+        return serverType;
+    }
+
+    public static Optional<DatabaseType> build(String serverType) {
```

Comment:
Is the internal string serverType and `build` really necessary? Why is it not enough to use the `valueOf` method?